### PR TITLE
The Nanotrasen Representative (MY WAY)

### PR DESCRIPTION
**Pull request by Gandalf2k15**
## The Nanotrasen Representative
image

The Nanotrasen Representative - Nanotrasen's finest, the first port of call for any bureaucratic incidents and above command support. 

The function of this role is advisory to command and you should mostly be concerned with command operations, ensuring they follow all relevant procedures. You should also be prioritizing Nanotrasens's goals and interests, making sure command follow these to a T. You are there to prevent malpractice and insubordination, ensure the command structure operates as per standard operating procedures, and most of all, you are there to ensure command follows the command structure.

Central command have authorised you to use your powers to ensure that command follows the structure that is set out. Insubordination of any kind will not be tolerated. You must also deal with any disputes or issues regarding command and the Captain, and are to contact Central Command should the need arise.

You may also be required to be an impartial mediator in trials, typically serving as legal counsel and defence for an accused crew member.

You, as the Nanotrasen Representative, do not have any real power aboard, though, cases where you manage to get Central Command's involvement change this. See your SOP for more information.

For now, we will likely be using the paradise wiki SOP: https://paradisestation.org/wiki/index.php/Nanotrasen_Representative

Your office is located in the interlink central command area.

image

**This role is veteran crew only.**

They are armed.

https://www.youtube.com/watch?v=qQzdAsjWGPg

Credits for the sprites go to ERIS, heavily modified by myself.

## Other changes
I've fixed up a few things regarding alt titles.

A new department(central command) has been added, and any special central command roles will go there. Starting with this and the Blueshield. It's time to start moving towards roleplay!

## Changelog
:cl:
add: The Nanotrasen Representative has been added to the game!
/:cl:

### CODE REVIEW

**Comment by CameronWoof** [comment]
I really like the idea of this role, and while I would have liked to see this be an update for Lawyer, the role existing at all is certainly better than it not.

Can NT Representatives be antagonists?

**Comment by Gandalf2k15** [comment]
> I really like the idea of this role, and while I would have liked to see this be an update for Lawyer, the role existing at all is certainly better than it not.
> 
> Can NT Representatives be antagonists?

Not sure if we should allow them to be antags.

**Comment by FlamingLily** [comment]
> > I really like the idea of this role, and while I would have liked to see this be an update for Lawyer, the role existing at all is certainly better than it not.
> > Can NT Representatives be antagonists?
> 
> Not sure if we should allow them to be antags.

They almost certainly shouldn't be, imho.

**Comment by CameronWoof** [comment]
I think it interferes with their station as a trusted mediator (especially with the authority they have) if they can be antagonists--but also, the idea of an antagonist spawning on the interlink seems very iffy.

**Comment by softcerv** [comment]
Sweet, it's going to be nice to have a job that can better help deal with bad command staff.

**Comment by FlamingLily** [comment]
Personally, I still think it would be better to have two roles for this, one to handle security issues and one to handle command issues, but either way this is gooood

**Comment by Arctaisia** [comment]
Very very nice, gonna give a kiss to you...

**Comment by YakumoChen** [comment]
I dont feel this really does much but be a lawyer with slightly more sway. CC still needs involved with any (acting) captain vs command disputes and admins typically are loathe to actually respond to messages sent to CC.

I have a feeling this is going to make any demotions become extremely obnoxious when the crew just gets the rep involved.

**Comment by Gandalf2k15** [comment]
> I dont feel this really does much but be a lawyer with slightly more sway. CC still needs involved with any (acting) captain vs command disputes and admins typically are loathe to actually respond to messages sent to CC.
> 
> I have a feeling this is going to make any demotions become extremely obnoxious when the crew just gets the rep involved.

We have discussed with admins that they aren't to be ignored.(still to add the admin message system) It's an RP first role, I know that sounds hard.

**Comment by YakumoChen** [comment]
> 
> We have discussed with admins that they aren't to be ignored.(still to add the admin message system) It's an RP first role, I know that sounds hard.

The issues I mentioned above already involve a lot of RP, and need admin involvement (IC or not) when they go above that, not another role that serves the same niche as Lawyer but with a gun.

**Comment by YakumoChen** [comment]
If this role actually gets admins to resolve disputes between heads of staff though then by all means I'm for it. The track record I've seen as a very frequent command player though, is that ahelps have a much better response rate than a fax to Centcomm

**Comment by YakumoChen** [comment]
> It is also your duty as an NT Rep to ensure that you are appraised of any active threat aboard the station, and subsequently inform Central Command of its presence if they are unaware of the situation at hand. Whether or not you are proactive within accurately and aptly reporting on active threats aboard the station can be the difference between receiving an Amber, Red, or Gamma ERT in an emergency

I am also concerned this is going to lead to a lot more metaknowlege begging (faxing CC in order to get lings incinerated before they do anything) or purely validhunting ERTs sent more frequently than they need to.

**Comment by dankmemerino147** [comment]
> 
> 
> If this role actually gets admins to resolve disputes between heads of staff though then by all means I'm for it. The track record I've seen as a very frequent command player though, is that ahelps have a much better response rate than a fax to Centcomm

as staff, i will say that this is because CC faxes either don't make noise or make the most horrid printer noise so everyone mutes it. personally whenever i fax CC as command i just ahelp "cc fax" or something like that to poke the admins' ears about it

**Comment by YakumoChen** [comment]
> 
> as staff, i will say that this is because CC faxes either don't make noise or make the most horrid printer noise so everyone mutes it. personally whenever i fax CC as command i just ahelp "cc fax" or something like that to poke the admins' ears about it

as a player I wish this was made more clear because I would have absolutely never known this. One would think the IC centcomm message preclude the need for an ahelp.

**Comment by ghost** [comment]
Makes me wonder if it's worth waiting on the fax machine PR to finally be finished before this comes in. Might be possible that their faxes get a special notice to admins too..

Other than that though, this is actually a really nice idea - though I'm a bit torn on the hat being beige lmao

**Comment by hugmeorelse** [comment]
Why not call this Nanotransen Magistrate? 👀

**Comment by Gandalf2k15** [comment]
> Why not call this Nanotransen Magistrate? 👀

Making a separate role for this

**Comment by Gandalf2k15** [comment]
> Makes me wonder if it's worth waiting on the fax machine PR to finally be finished before this comes in. Might be possible that their faxes get a special notice to admins too..
> 
> Other than that though, this is actually a really nice idea - though I'm a bit torn on the hat being beige lmao

I'll be adding a separate system for it, one on a computer. So admins can't ignore it.

**Comment by Gandalf2k15** [comment]
> > We have discussed with admins that they aren't to be ignored.(still to add the admin message system) It's an RP first role, I know that sounds hard.
> 
> The issues I mentioned above already involve a lot of RP, and need admin involvement (IC or not) when they go above that, not another role that serves the same niche as Lawyer but with a gun.

Laywers do nothing here. Never ever seen a lawyer demote someone. Namely because they have no authority and are just a meme role.

**Comment by Gandalf2k15** [comment]
image

**Comment by NullDagaf** [comment]
Uh-huh.
My assumption is that they're not "supposed" (can, but in most cases shouldn't) to go onto the station, considering the off-station office?
(Actually, fuck it, edit-in rant time.)
Also as a sidenote. People **won't** "act" like a real Nanotrasen Representative would. By that, I mean in most cases it'd devolve into unprofesionalism, , similarly to *some certain heads of staff*. And the few people that would act professional, would probably stop playing the role due to the lack of content it has. Yes, you can "ensure the command structure stays intact", yes, you're the mediator in any potential conflicts the station might have. But conflicts aren't omni-present and you can't stretch yelling at someone to take 2+ hours. And a CentCom, for the lack of a better word, "Official" wouldn't get sloshed on the job, now would they?

Before anyone drags this "amazing" idea in, no, enforcing professionality via rules and admins won't work, unless you set a very **very** high standard right away. The level of professionalism a potential CentCom representative holds is so high, trying to correct it would only make the enforcing admins seem like nitpickers and huge dicks. And conversely, if you don't hold the job to a high standard, the players (generalizing here) will complain about how "the rep acts like an idiot/leaves to ERP during an emergency", and potentially, some admins could too - wanting the role to act more professionally, and wanting to correct players' behavior, but looking biased if they do due to the already low standard the job has.

**Comment by Gandalf2k15** [comment]
> Uh-huh.
> My assumption is that they're not "supposed" (can, but in most cases shouldn't) to go onto the station, considering the off-station office?
> (Actually, fuck it, edit-in rant time.)
> Also as a sidenote. People **won't** "act" like a real Nanotrasen Representative would. By that, I mean in most cases it'd devolve into unprofesionalism, , similarly to _some certain heads of staff_. And the few people that would act professional, would probably stop playing the role due to the lack of content it has. Yes, you can "ensure the command structure stays intact", yes, you're the mediator in any potential conflicts the station might have. But conflicts aren't omni-present and you can't stretch yelling at someone to take 2+ hours. And a CentCom, for the lack of a better word, "Official" wouldn't get sloshed on the job, now would they?
> 
> Before anyone drags this "amazing" idea in, no, enforcing professionality via rules and admins won't work, unless you set a very **very** high standard right away. The level of professionalism a potential CentCom representative holds is so high, trying to correct it would only make the enforcing admins seem like nitpickers and huge dicks. And conversely, if you don't hold the job to a high standard, the players (generalizing here) will complain about how "the rep acts like an idiot/leaves to ERP during an emergency", and potentially, some admins could too - wanting the role to act more professionally, and wanting to correct players' behavior, but looking biased if they do due to the already low standard the job has.

Tldr, see para policy

**Comment by Arctaisia** [comment]
> > Why not call this Nanotransen Magistrate? 👀
> 
> Making a separate role for this

Gandalf, you're spoiling us :heart:
